docker-compose.yml

```diff
@@ -128,8 +128,6 @@ services:
     sysctls:
       - net.ipv6.conf.all.disable_ipv6=0
     depends_on:
-      gateway:
-        condition: "service_healthy"
```

Comment:
This is so the client could be started before the gateway in an upcoming test?

Reply:
Yeah, removing this so we can explicitly test it as part of #2022